document_page_access_group_user_role/migrations/16.0.1.1.0/post-migration.py

```diff
@@ -0,0 +1,12 @@
+# Copyright 2024 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    """Pages that had roles should now have the correct users."""
+    pages = env["document.page"].sudo().search([("role_ids", "!=", False)])
+    for page in pages:
+        page.role_ids = False
```

Comment:
Why emptying roles?

Reply:
Ok sorry, it's already changed. You get the users that you will have to have linked (to define them later) and then you empty the groups. This is to avoid the constrains _check_document_page_groups_users_